modules/dkan/dkan_periodic_updates/dkan_periodic_updates.module

```diff
@@ -408,6 +416,91 @@ function dkan_periodic_updates_state() {
   return $output;
 }
 
+
+/**
+ * Page callback for periodic updates reset page.
+ *
+ * @param $form
+ * @param $form_state
+ *
+ * @return mixed
+ */
+function dkan_periodic_updates_reset_form($form, &$form_state) {
+  $active = variable_get('dkan_periodic_updates_status');
+  if ($active) {
+    $file = variable_get('dkan_periodic_updates_manifest');
+    $file = file_load($file);
+    if ($file) {
+      $handle = fopen($file->uri, 'r');
+      $manifest_options = [];
+      $headers = fgetcsv($handle, 0, ',');
+      while (($data = fgetcsv($handle, 0, ",")) !== FALSE) {
+        $resource_data = array_combine($headers, $data);
+        if (!empty(variable_get('dkan_periodic_updates_' . $resource_data['resource_id'], ''))) {
+          $manifest_options[$resource_data['resource_id']] = $resource_data['file_url'];
```

Comment:
Can we make the options a little bit more verbose? Right now it is just the url, maybe we can have a string with the url and the resource ID? Though, it is up to you if we do this, we can discuss in scrum.

Reply:
As we talk, is easier for the user to look by url and adding the resource ID would make the text too long.